### PR TITLE
update: increase the max length of the secret (#3493)

### PR DESCRIPTION
**Pull request by rambhosale**
This PR increases the default maxlength from 1024 characters to 10000, which should be good for most cases.

### CODE REVIEW

File: web/src/components/NewSecretModalButton/NewSecretModalButton.tsx [hunk 1]
```diff
@@ -144,6 +144,7 @@ export const NewSecretModalButton: React.FC<NewSecretModalButtonProps> = ({
                       tooltipProps={{
                         dataTooltipId: 'secretDescriptionTextField'
                       }}
+                      maxLength={5000000}
```
Comment (tan-nhu) [inline web/src/components/NewSecretModalButton/NewSecretModalButton.tsx:147]: Thanks @rambhosale for the PR. Can you move this number to `web/src/Config.ts`, name it something like `SECRET_LIMIT_IN_BYTES` or something like that, and use it here?

Also it stead of `5000000`, let's make it `5_242_880`?

Comment (rambhosale) [reply web/src/components/NewSecretModalButton/NewSecretModalButton.tsx]: Hi @tan-nhu I have extracted the length to the config file and also made a similar change in the secrets update section, that was missed last time

Comment (tan-nhu) [reply web/src/components/NewSecretModalButton/NewSecretModalButton.tsx]: Thank you for making the change. The UI code looks good. I'll sync with my team tomorrow to see if any backend is needed to support increasing this secret length. Will follow up.

Comment (tan-nhu) [reply web/src/components/NewSecretModalButton/NewSecretModalButton.tsx]: Confirmed there's no extra backend change needed to support this.

File: web/src/components/NewSecretModalButton/NewSecretModalButton.tsx [hunk 2]
```diff
@@ -38,6 +38,7 @@ import { useStrings } from 'framework/strings'
 import type { OpenapiCreateSecretRequest, TypesSecret } from 'services/code'
 import { getErrorMessage } from 'utils/Utils'
 import css from './NewSecretModalButton.module.scss'
+import Config from 'Config'
```
Comment (tan-nhu) [inline web/src/components/NewSecretModalButton/NewSecretModalButton.tsx:41]: @rambhosale Please move this import above the css import to fix linting.

File: web/src/components/UpdateSecretModal/UpdateSecretModal.tsx [hunk 3]
```diff
@@ -40,6 +40,7 @@ import type { OpenapiUpdateSecretRequest, TypesSecret } from 'services/code'
 import type { SecretFormData } from 'components/NewSecretModalButton/NewSecretModalButton'
 import { getErrorMessage, truncateString } from 'utils/Utils'
 import css from './UpdateSecretModal.module.scss'
+import Config from 'Config'
```
Comment (tan-nhu) [inline web/src/components/UpdateSecretModal/UpdateSecretModal.tsx:43]: Same here

**Comment by xzycn** [comment]
Hi, there is any progress on this? This very import issue.
We are painful to store k8s cert.

**Comment by tan-nhu** [comment]
There's still an error with `prettier`. I merged anyway. I'm fixing the issue from our end. Thanks @rambhosale, @xzycn for the contribution.

**Comment by xzycn** [comment]
@tan-nhu 
Hi, thx for your job.
May I ask when your next release will come? I have recently use gitness([v3.0.0-beta.6](https://github.com/harness/gitness/releases/tag/v3.0.0-beta.6)) as our CI/CD tools,there are some bugs or inconvenient design.
So I want a newer release including this PR :)

**Pull request merged**